app/src/main/res/layout/project_layout.xml

```diff
@@ -46,4 +53,66 @@
       app:backgroundTint="@color/button_secondary" />
   </RelativeLayout>
 
-</RelativeLayout>
+  <androidx.cardview.widget.CardView
+    android:id="@+id/rewards_container"
+    android:layout_width="match_parent"
+    android:layout_height="0dp"
+    android:layout_alignParentBottom="true"
+    app:cardBackgroundColor="@color/ksr_grey_500"
+    app:cardCornerRadius="@dimen/grid_5"
+    app:layout_constrainedHeight="false"
+    app:layout_constraintTop_toTopOf="@+id/guideline">
+
+    <LinearLayout
+      android:id="@+id/pledge_container"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:alpha="0"
+      android:orientation="vertical">
+
+      <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/rewards_appbar"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:elevation="0dp">
+
+        <androidx.appcompat.widget.Toolbar
+          android:id="@+id/rewards_toolbar"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:background="@color/ksr_grey_500"
```

Comment:
this color is different from the background color of the rewards fragment. i think from the abstract designs they should be the same.